app/src/main/java/com/el/yello/presentation/onboarding/fragment/code/CodeFragment.kt

```diff
@@ -95,37 +95,49 @@ class CodeFragment : BindingFragment<FragmentCodeBinding>(R.layout.fragment_code
     }
 
     private fun initIdEditTextViewError() {
-        binding.etCode.setBackgroundResource(R.drawable.shape_fill_red20_line_semantic_status_red500_rect_8)
-        binding.ivCodeDelete.setBackgroundResource(R.drawable.ic_onboarding_delete_red)
-        binding.tvCodeWarning.text = getString(R.string.onboarding_code_duplicate_msg)
-        binding.tvCodeWarning.setTextColor(
-            ContextCompat.getColor(
-                requireContext(),
-                R.color.semantic_red_500,
-            ),
-        )
+        with(binding) {
+            etCode.setBackgroundResource(R.drawable.shape_fill_red20_line_semantic_status_red500_rect_8)
+            ivCodeDelete.setBackgroundResource(R.drawable.ic_onboarding_delete_red)
+            tvCodeWarning.text = getString(R.string.onboarding_code_duplicate_msg)
+            tvCodeWarning.setTextColor(
+                ContextCompat.getColor(
+                    requireContext(),
+                    R.color.semantic_red_500,
+                ),
```

Comment:
```suggestion
                colorOf(R.color.semantic_red_500),
``` 
요친구 확장함수 활용하면 더 깔끔해져요 ~~
저렇게 했는데 안뜨면 앞에 requireContext().colorOf 였나..

Reply:
헐 맞다 까묵고 있었네욥 오케이 ~~~!